web-module/src/main/webapp/resources/structEditJS/StructEditor.mjs

```diff
@@ -798,17 +798,26 @@ window.CELEMENTS.structEdit.CelementsFormSaver = Class.create({
     _me._saveCallback = saveCallback;
   },
 
-  _saveAndContinueAjax : function(formName, handler) {
-    var _me = this;
-    if(document.forms[formName]) {
-      document.forms[formName].select('textarea.mceEditor').each(function(formfield) {
-         console.log('textarea save tinymce: ', formfield.name, tinyMCE.get(formfield.id).save());
-         formfield.value = tinyMCE.get(formfield.id).save();
-       });
-       $(formName).request(handler);
-     } else {
-       console.error('form not found: ', formName);
-     }
+  _saveAndContinueAjax : function(formId, options) {
+    const form = document.forms[formId];
+    if (form) {
+      form.querySelectorAll('textarea.mceEditor').forEach(textarea => {
+        console.debug('textarea save tinymce: ', textarea);
+        textarea.value = tinyMCE.get(textarea.id).save();
+      });
+      options.parameters = options.parameters || {};
+      // submit enabled select fields without selections
+      [...form.querySelectorAll('select')]
+        .filter(select => select.name && !select.disabled && select.value === '')
+        .forEach(select => options.parameters[select.name] = select.dataset.unselectedValue || '');
+      // submit unchecked checkboxes and radio buttons
+      [...form.querySelectorAll('input[type="checkbox"], input[type="radio"]')]
+        .filter(input => input.name && !input.disabled && !input.checked)
+        .forEach(input => options.parameters[input.name] = input.dataset.uncheckedValue || '');
+      form.request(options);
```

Comment:
`form.request` has only been introduced in Safari iOS 14 in April 2021. https://caniuse.com/?search=form.request
Moreover it is not getting transpiled by babel.

It therefore relies on the prototypejs implementation:
http://api.prototypejs.org/dom/Form/request/index.html 

Can we get rid of the prototypejs dependency by using [FormData and fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#uploading_a_file) instead?

Reply:
refactoring the previously employed prototypejs `form.request` to vanillajs `fetch` is not in the scope of CELDEV-1205